give.php

```diff
@@ -326,6 +337,7 @@ private function includes() {
 			require_once GIVE_PLUGIN_DIR . 'includes/actions.php';
 			require_once GIVE_PLUGIN_DIR . 'includes/filters.php';
 			require_once GIVE_PLUGIN_DIR . 'includes/api/class-give-api.php';
+			require_once GIVE_PLUGIN_DIR . 'includes/class-notices.php';

```

Comment:
@ravinderk Just a thought whether the `Give_Notices` class should be in admin folder as the notices will be displayed in admin only and not in front end. So, I think the filename and location of the file should be as it was earlier `includes/admin/class-admin-notices.php`. Let me know your thoughts.